src/js/gi/components/SearchResult.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import Estimator from '../utils/estimator';
+
+class SearchResult extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.renderCautionFlag = this.renderCautionFlag.bind(this);
+    this.renderName = this.renderName.bind(this);
+    this.renderLocality = this.renderLocality.bind(this);
+    this.renderVetCount = this.renderVetCount.bind(this);
+    this.renderTuition = this.renderTuition.bind(this);
+    this.renderHousing = this.renderHousing.bind(this);
+    this.renderBooks = this.renderBooks.bind(this);
+    this.estimate = this.estimate.bind(this);
+  }
+
+  // WIP
+  estimate() {
+    // get current values about the user
+    const e = this.props.estimator;
```

Comment:
I see that it's a WIP, but in case you didn't already know, it's not considered good practice to modify props.